generate-wayland.sh

```diff
@@ -4,7 +4,26 @@ set -ex
 # Generates Wayland client protocol sources needed for e.g. GLFW
 # https://github.com/glfw/glfw/blob/6281f498c875f49d8ac5c5a02d968fb1792fd9f5/src/CMakeLists.txt#L98-L118
 
-# TODO: actually generate wayland-protocol sources
+gen_dir="root/usr/share/wayland-generated"
+protocols_dir="root/usr/share/wayland-protocols"
```

Comment:
This would effectively become just `protocols_dir="wayland-protocols"` - in fact maybe we don't even need a var for it.